scripts/transpile.js

```diff
@@ -0,0 +1,11 @@
+const { transpileFiles } = require('@asyncapi/generator-react-sdk');
+const path = require('path')
+
+async function transpileTemplate() {
+  const templateContentDir = path.join(__dirname, '../template');
+  console.log("templateContentDir",templateContentDir)
+  const outputDir = path.join(__dirname, '../__transpiled');
+  await transpileFiles(templateContentDir, outputDir, {recursive: true})
```

Comment:
please put it in between try/catch instead of later catching errors on transpileTemplate